source/tasks/Utils/install.ts

```diff
@@ -94,7 +94,10 @@ async function getOrDownloadOcto(option: DownloadOption, download?: (option: Dow
 }
 
 async function resolvePublishedOctoVersion(version?: string): Promise<DownloadOption> {
-    console.log(`Attempting to contact ${OctopurlsUrl} to find latest CLI tools`);
+    if (version === null || version === undefined) {
+        version = "latest";
+    }
+    console.log(`Attempting to contact ${OctopurlsUrl} to find octo version ${version}`);
```

Comment:
is `octo` here the clearest we can make it? The repo is called `OctopusCLI`... What do we call it on the download page?

Reply:
Good point. I've run through all the messages I could find, standardizing on "Octo command line tool", and adding some missing "the". I've started omitting the `.exe` because it seems less friendly to me, and not actually accurate as we move towards .NET Core.